core/db/migrate/20151013222921_remove_spree_tokenized_permissions.rb

```diff
@@ -0,0 +1,14 @@
+class RemoveSpreeTokenizedPermissions < ActiveRecord::Migration
+  def up
+    drop_table :spree_tokenized_permissions
+  end
+
+  def down
+    create_table "spree_tokenized_permissions" do |t|
+      t.integer  "permissable_id"
+      t.string   "permissable_type"
+      t.string   "token"
+      t.timestamps
```

Comment:
if anything it should be `null: true` to generate the same as the existing table